data/pages-content/thankyou.md

```diff
@@ -1,368 +1,378 @@
+- Vaibhav (VB) Srivastav
+- Cyril Bitterich
+- Rodrigo Girão Serrão
 - Diego Russo
-- Ernesto Arbitrio
-- Nick Radcliffe
-- Patrick Arminio
+- Nicholas Tollervey
+- Cristián Maureira-Fredes
+
+### [Speaker Mentorship Programme](/mentorship)
+
 - Theofanis Petkos
+- Diego Russo
+<!-- - FIXME - Ask Fanis if we are missing people. -->
+
+### Programme Reviewers
 
 The talk selection process was supported by our wonderful reviewers:
 
-- Alexander CS Hendorf
-- Alisa Dammer
-- Andy Piper
-- Anmol Krishan Sachdeva
-- Aurélie Vancraeyenest
+- Agata Migalska
+- Alexander Hess
+- Alisa
+- Andrii Soldatenko
+- Christian Ledermann
+- Clinton Roy
 - Cosima Meyer
+- Cristián Maureira-Fredes
+- Cyril Bitterich
+- David Seddon
+- David Vaz
+- Deborah Leem
 - Diego Russo
 - Ernesto Arbitrio
-- Francesco Tisiot
 - Gajendra Deshpande
-- Guillermo Navas-Palencia
-- Henrique Leal
-- Janit Kumar Anjaria
-- Laysa Uchoa
-- Luca Corbucci
-- Luka Zeleznik
-- Marco Richetta
+- Jiawei Wang
+- Lisa Carpenter
+- Manav Mehra
+- Maxim Danilov
 - Nabanita Roy
-- Nicholas H. Tollervey
-- Nick Radcliffe
-- Nicola Melluso
-- Pav Andriychenko
-- Pradhvan Bisht
-- Sean Corcoran
-- Stephanie Rideout
-- Vaibhav Srivastav (VB)
+- Nicholas Tollervey
+- nikkie
+- Pooja Gadige
+- Pratibha Jagnere
+- Radomir Dopieralski
+- Raghav Dave
+- Richard Kellner
+- Roberto Polli
+- Sebastiaan Zeeff
+- Sebastian Witowski
+- Shagun Sodhani
+- Siddharth Gupta
+- Tiago Montes
+- Valerio Maggio
+- Vincent Poulailleau
+- Vinícius Gubiani Ferreira
 
 ## Operations Team
 
 From managing recording & streaming, to liaising with our wonderful remote and
-on-site volunteers, the Operations team have all covered. If you saw someone
-with a walkie-talkie or received a reply from our helpdesk, chances are you met
-someone from this team :)
+on-site volunteers, the Operations team have all covered. If you ever received a reply from our helpdesk, chances are you interacted with someone from this team :)
 
-- Nicolás Demarchi
 - Theofanis Petkos
-- Anmol Krishan Sachdeva
-- Artur Czepiel
-- Francesco Pierfederici
+- Cyril Bitterich
+- Nicolás Demarchi
+- Dhanshree Arora
+- Shekhar Koirala
+- Artem Kislovskiy
 - Martin Borus
-- Raquel Dou
-- Vaibhav Srivastav (VB)
-- Yasmin Morshed
+- Daksh P. Jain
+- Sebastiaan Zeeff
+- Cristián Maureira-Fredes
 
-## Sponsors Team
+### EuroPython 2023 Discord Team
 
-Most of the sponsorship contacts, logistics & the exhibits are managed mostly by
-the superwoman Silvia. Here are the small team that helped with sponsorship
-management:
+The conference discord community structure and the programme notification bot was created by:
+- Sebastiaan Zeeff
 
-- Silvia Uberti (the superwoman of the team)
-- Raquel Dou
-- Sayan Chowdhury
+The registration discord bot was developed and deployed by:
+- Artem Kislovskiy
+- Sebastiaan Zeeff
+- Cheuk Ting Ho
+- Cristián Maureira-Fredes
+- Kasisnu Singh
+- Dhanshree Arora
+- Martin Borus
+<!-- - FIXME - Ask Artem if I am missing someone. -->
```

Comment:
All good, @artcz do you want to be in the list 😀?

Reply:
I forgot to add Artur. Fixed now here: https://github.com/EuroPython/website/pull/396/commits/d1674762380730a38ef6bed15a3debc8c42aab57